packages/message-list/src/vaadin-message-list-mixin.js

```diff
@@ -47,6 +47,17 @@ export const MessageListMixin = (superClass) =>
           observer: '__markdownChanged',
           reflectToAttribute: true,
         },
+
+        /**
+         * When set to `true`, new messages are announced to assistive technologies using ARIA live regions.
+         * @type {boolean}
```

Comment:
Let's also add `@attr {boolean} announce-messages` here and in `.d.ts`.

Reply:
Done